packages/client/src/client.js

```diff
@@ -406,10 +406,26 @@ export class PercyClient {
     });
   }
 
-  async uploadComparisonTile(comparisonId, { index = 0, total = 1, filepath, content } = {}) {
+  async uploadComparisonTile(comparisonId, { index = 0, total = 1, filepath, content, sha } = {}) {
     validateId('comparison', comparisonId);
     this.log.debug(`Uploading comparison tile: ${index + 1}/${total} (${comparisonId})...`);
     if (filepath) content = await fs.promises.readFile(filepath);
+    if (sha) {
+      let retries = 10;
+      setTimeout(() => this.log.debug(`Verifying comparison tile with sha: ${sha}`), 1000);
```

Comment:
whats the the reason for `setTimeout` here?

Reply:
added an initial sleep so that our avg number for retries is low, 1st call would return true n most cases when we provide 1s sleep.

Reply:
a correct way to sleep would be awaiting on the promise something like [this](https://github.com/percy/cli/blob/54fa2c10eea66c2bdaec3c540d46ecce4b1d5acc/packages/core/src/utils.js#L149-L166)].